gettsim/tests/test_data/test_dfs_eltg.csv

```diff
@@ -0,0 +1,13 @@
+hid,tu_id,pid,m_wage,m_wage_l1,east,incometax,soli,svbeit,elt_zeit,geschw_bonus,year,,,elt_geld,considered_wage,payed_percentage
+1,1,23,0,0,FALSE,0,0,0,FALSE,FALSE,2018,,,0,,
```

Comment:
do we need `incometax`, `soli` `svbeit` here or aren't they calculated by the function? In addition, `geschw_bonus` needs to be made endogenous.

Reply:
Yes. For the net income of the current year! Or is this all calculated by these proxy parameters? `geschw_bonus` is defnetly endogenous as well as `elt_zeit`. Just was not sure if we touch eligibility here. As I am now, I will include.

Reply:
Right. As soon as you're done, I'll add my own elterngeld calculation as a test ;)